lib/fileutils.rb

```diff
@@ -1863,7 +1869,9 @@ def fu_get_gid(group)   #:nodoc:
   # Updates modification times (mtime) and access times (atime)
   # of the entries given by the paths in +list+
   # (a single path or an array of paths);
-  # by default, creates an empty file for any path to a non-existent entry.
+  # returns +list+ if it is an array, <tt>[list]</tt> otherwise.
+  #
+  # By default, creates an empty file for any path to a non-existent entry.
```

Comment:
I think it would be a good idea to mention that the `nocreate`  keyword argument can change this behavior, otherwise the reader might be confused as to what "non-default" means.

Reply:
Mentioned.